src/Nav/index.js

```diff
@@ -21,6 +21,14 @@ export default function Nav({ mobile = false }) {
         >
           Features
         </Link>
+        <Link
+          href="https://katacoda.com/dvc/courses/basics"
```

Comment:
Again, we can't do this. It's not scalable. It might become part of the resources block in the community page that is under development, or might become at some point a drop-down menu. Let's not do this for now.

Reply:
I removed it for the time being, we can add it again later, if needed.